client/landing/stepper/index.tsx

```diff
@@ -49,12 +55,32 @@ function initializeCalypsoUserStore( reduxStore: any, user: CurrentUser ) {
 	reduxStore.dispatch( requestSites() );
 }
 
-const FlowWrapper: React.FC< { user: UserStore.CurrentUser | undefined } > = ( { user } ) => {
+interface configurableFlows {
+	flowName: string;
+	pathToFlow: Flow;
+}
+
+const availableFlows: Array< configurableFlows > = [
+	{ flowName: 'tailored', pathToFlow: tailoredSignup },
+	{ flowName: 'newsletters', pathToFlow: newsletters },
+	{ flowName: 'link-in-bio', pathToFlow: linkInBio },
+	{ flowName: 'podcasts', pathToFlow: podcasts },
+];
+
+const FlowSwitch: React.FC< { user: UserStore.CurrentUser | undefined } > = ( { user } ) => {
 	const { anchorFmPodcastId } = useAnchorFmParams();
+	const flowName = useQuery().get( 'flow' );
+
 	let flow = siteSetupFlow;
 
 	if ( anchorFmPodcastId ) {
 		flow = anchorFmFlow;
+	} else {
+		availableFlows.find( ( currentFlow ) => {
```

Comment:
Hmmm, question.

Can we have more than one flow per time ?  I mean, can It be `podcasts and newsletter` at the same time?

if we use `Find` we remove the necessity of looping over the whole array to get the flow, once found the matching name we stop looping, which might be faster.

if we use `forEach` we will loop over the whole array.

Reply:
Well, that's right, but you're looping the whole array in your original version as well, because find will only stop if you return truthy, and the original code never did.